### PR TITLE
add github templates for issues and pull requests

### PR DESCRIPTION
**Pull request by ahnpnl**
Follow Jest templates :smile:

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/question.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+---
+
+## 💬 Questions and Help
+
+### Please note that this issue tracker is not a help forum and this issue will be closed.
+
+For questions or help please see:
+
+- [The jest-preset-angular online documentation](https://github.com/thymikee/jest-preset-angular/blob/master/README.md)
+- The [jest-preset-angular](https://stackoverflow.com/questions/tagged/jest-preset-angular) tag on [StackOverflow](https://stackoverflow.com/questions/ask)
```
Comment (wtho) [inline .github/ISSUE_TEMPLATE/question.md:14]: For now we were pretty open for help questions, are we going to restrict more in the future?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+## Does this PR introduce a breaking change?
+
+- [ ] Yes
+- [ ] No
+
+<!-- If this PR contains a breaking change, please describe the impact and migration plan -->
+
+
+## Other information
```
Comment (wtho) [inline .github/PULL_REQUEST_TEMPLATE.md:21]: I think we should also mention here that an entry for the Changelog is desirable

File: .github/ISSUE_TEMPLATE/question.md (hunk 1 above)
Comment (ahnpnl) [reply .github/ISSUE_TEMPLATE/question.md:14]: The template for question is more like just for structuring issues. 

Maybe the instruction text can be changed in such a way that it is not mandatory to check on StackOverflow ?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (ahnpnl) [reply .github/PULL_REQUEST_TEMPLATE.md:21]: agree

Comment (wtho) [reply .github/PULL_REQUEST_TEMPLATE.md:21]: Maybe more explicit:
```md
## Changelog
- [ ] I added an entry to `CHANGELOG.md`

<!-- Please add an entry to CHANGELOG.md, summarizing the change of this PR and add them to the according section under "master": -->
<!-- * <short-summary> ([#<pr-nr>](https://github.com/thymikee/jest-preset-angular/pull/<pr-nr>)) -->
```

Comment (ahnpnl) [reply .github/PULL_REQUEST_TEMPLATE.md:29]: I like that one, already added in

File: .github/ISSUE_TEMPLATE/question.md (hunk 1 above)
Comment (wtho) [reply .github/ISSUE_TEMPLATE/question.md:14]: Actually I am ok with moving issues towards SO, but I would have to subscribe to our tag then :sweat_smile:

Comment (ahnpnl) [reply .github/ISSUE_TEMPLATE/question.md:14]: I think we should use both channels, GitHub and SO. Maybe the text can be something like:

“For questions or help, you can also check on StackOverflow..” ?

**Comment by wtho** [comment]
I created [a filter on SO](https://stackexchange.com/filters/393541/jest-preset-angular) and subscribed to it, just in case. You can also just subscribe to it and select how often you want to be notified.
Anyway jest-preset-angular is not very active on SO.

@thymikee can you have a look? We should get this merged!

File: .github/ISSUE_TEMPLATE/question.md (hunk 1 above)
Comment (ahnpnl) [reply .github/ISSUE_TEMPLATE/question.md]: I modified this text to the one I mentioned above.

**Comment by wtho** [comment]
@ahnpnl if you think this PR is ready, we can merge it. Looks good to me.

File: .github/ISSUE_TEMPLATE/question.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+name: 💬 Questions / Help
+labels: ':speech_balloon: Question'
+about: If you have questions, please check StackOverflow
+---
+
+## 💬 Questions and Help
+
+### Please note that this issue tracker is not a help forum and this issue will be closed.
+
+For questions or help, you can also check on StackOverflow https://stackoverflow.com/questions/tagged/jest-preset-angular) tag on [StackOverflow](https://stackoverflow.com/questions/ask
```
Comment (ahnpnl) [inline .github/ISSUE_TEMPLATE/question.md:11]: I gonna correct this to comply with markdown syntax and rebase master then it’s ready to merge

**Pull request merged**